setup.py

```diff
@@ -30,7 +30,7 @@ def get_requirements():
     long_description=get_long_description(),
     long_description_content_type="text/markdown",
     classifiers=[
-        "Development Status :: 3 - Alpha",
+        "Development Status :: 4 - Beta",
```

Comment:
Should be 5 - Stable no? If we are moving to 1.0 it means that is stable

Reply:
Not sure.  To me, 1.0.0 means that we are committing to doing semver-compatible version bumping from this point, e.g. any backwards-incompatible API changes will require the version to be bumped to 2.0.0.  But that's just about the API.  I don't have much evidence yet that the implementation actually works and is bug-free, so it feels premature to call it Stable.